docs/sphinx/index.rst

```diff
@@ -65,6 +65,12 @@ Example Usage
         print("%(timestamp)s %(author)s: %(text)s" % hit["_source"])
 
 
+Interactive examples
+-------------------
```

Comment:
Oops, I missed this:

```suggestion
--------------------
```

Reply:
Thanks! ::Shakes fist at reStructuredText::